takwimu/templates/takwimu/_includes/navbar/about.html

```diff
@@ -5,68 +5,65 @@
 
 {% block dropdown_id %}nav-about{% endblock %}
 
-{% block dropdown_title %}About Takwimu{% endblock %}
-{% block dropdown_desc %}Takwimu provides quality insights and data-driven analysis to empower African development champions.{% endblock %}
+{% block dropdown_title %}<p class="text-right pr-3">About Takwimu</p>{% endblock %}
+{% block dropdown_desc %}<p class="text-right pr-3 font-weight-light">Takwimu provides quality insights and data-driven analysis to empower African development champions.</p>{% endblock %}
```

Comment:
Please make these changes in `takwimu/templates/takwimu/_includes/navbar/_dropdown.html` instead to apply style to all dropdowns. Not just this one.